STANDARD.md

```diff
@@ -191,7 +191,6 @@ standard:
     - description: an arbitrary comment
     - example: `After each time step we randomly removed 5 particles.`
 
-
```

Comment:
please keep the double newline, we have it before each headline to structure the document